src/js/components/histogram.coffee

```diff
@@ -0,0 +1,31 @@
+require '../../css/histogram.styl'
+
+React = require 'react'
+_ = require 'lodash'
+
+{div, h3, p, strong} = React.DOM
+
+Histogram = React.createClass
+  render: ->
+    className = 'histogram'
+    counts = @props.counts
+    colors = @props.colors
```

Comment:
Coffeescript (and ES6/TypeScript) tip: you can use `{counts, colors} = @props` here